rfcs/text/0030/process.yml

```diff
@@ -0,0 +1,310 @@
+        - type: match_only_text
+          name: text
+
+    - name: title
+      level: extended
+      type: keyword
+      short: Process title.
+      description: >
+        Process title.
+
+        The proctitle, some times the same as process name. Can also be different:
+        for example a browser setting its title to the web page currently opened.
+      multi_fields:
+        - type: match_only_text
+          name: text
+
+    - name: thread.id
+      format: string
+      level: extended
+      type: long
+      example: 4242
+      description: >
+        Thread ID.
+
+    - name: thread.name
+      level: extended
+      type: keyword
+      example: 'thread-0'
+      description: >
+        Thread name.
+
+    - name: start
+      level: extended
+      type: date
+      example: "2016-05-23T08:05:34.853Z"
+      description: >
+        The time this process started.
+
+    - name: ppid_start
+      level: extended
+      type: date
+      example: "2016-05-23T08:05:34.853Z"
+      description: >
+        The time this process's parent started.
+
+    - name: pgid_start
+      level: extended
+      type: date
+      example: "2016-05-23T08:05:34.853Z"
+      description: >
+        The time this process's process group started.
+
+    - name: sid_start
+      level: extended
+      type: date
+      example: "2016-05-23T08:05:34.853Z"
+      description: >
+        The time this process's session started.
+
+    - name: uptime
+      level: extended
+      type: long
+      example: 1325
+      description: >
+        Seconds the process has been up.
+
+    - name: working_directory
+      level: extended
+      type: keyword
+      example: /home/alice
+      description: >
+        The working directory of the process.
+      multi_fields:
+        - type: match_only_text
+          name: text
+
+    - name: exit_code
+      level: extended
+      type: long
+      example: 137
+      short: The exit code of the process.
+      description: >
+        The exit code of the process, if this is a termination event.
+
+        The field should be absent if there is no exit code for the event (e.g.
+        process start).
+
+    - name: end
+      level: extended
+      type: date
+      example: "2016-05-23T08:05:34.853Z"
+      description: >
+        The time the process ended.
+
+    - name: interactive
+      level: extended
+      type: boolean
+      example: true
+      short: Whether the process is connected to an interactive shell.
+      description: >
+        Whether the process is connected to an interactive shell.
+
+        Process interactivity is infered from the processes file descriptors. If the character device for the controlling tty is the same as stdin and stderr for the process, the process is considered interactive.
+
+        Note: A non-interactive process can belong to an interactive session. Backgrounded processes are an example of this.
+
+    - name: tty
+      level: extended
+      type: fd
```

Comment:
Are you trying to utilize `file_description.*` here? 

If so, you'll want to replace this with a re-use to `file_description.*`

```
 - at: process
   as: tty
   short_override: ...
```